test/input_gen/genInput.py

```diff
@@ -361,16 +363,17 @@ def bn_tf(x, *, trainable=True, init_beta=gen_tensor, init_gamma=gen_tensor, axi
     bnlayer = tf.keras.layers.BatchNormalization(
         axis=axis,
         trainable=trainable,
-        momentum=1.0,
+        momentum=0.99,
```

Comment:
Please use `0.9` for momentum for unittests. 
This increases the multiplier (0.01 to 0.1) which will make difference bigger (if any) and easier to identify

Reply:
Done